DuckDuckGo/it.lproj/Localizable.strings

```diff
@@ -1708,6 +1708,12 @@
 /* Title for the VPN Settings screen. */
 "network.protection.vpn.settings.title" = "Impostazioni VPN";
 
+/* Information message about New Tab Page redesign */
+"new.tab.page.intro.message.body" = "Personalizza i tuoi Preferiti e le funzionalità principali. Riorganizza gli elementi o nascondili per mantenere tutto in ordine.";
+
+/* Title of information message about New Tab Page redesign */
+"new.tab.page.intro.message.title" = "La tua pagina Nuova scheda è... Nuova!";
```

Comment:
Oh god, if all translations are like this one we sound a bit dumb XD The translation is not terrible but doesn't sound great either. This isn't this PR concern, we don't have a localisation review so no idea how to approach this. Let's ignore it for this PR's sake.